src/main/java/transforms/PrefixTransform.java

```diff
@@ -28,44 +28,59 @@
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.openlattice.shuttle.transformations.Transformation;
 import com.openlattice.shuttle.util.Constants;
+import org.apache.commons.lang3.StringUtils;
+
 import java.util.Objects;
 
-@JsonIgnoreProperties(value = {TRANSFORM} )
+@JsonIgnoreProperties( value = { TRANSFORM } )
 public class PrefixTransform extends Transformation<String> {
 
     private final String prefix;
 
     /**
      * Represents a transformation to add a prefix.
+     *
      * @param prefix: prefix to add
      */
     @JsonCreator
-    public PrefixTransform( @JsonProperty( Constants.PREFIX ) String prefix ) {
+    public PrefixTransform(
+            @JsonProperty( Constants.PREFIX ) String prefix ) {
         this.prefix = prefix;
     }
 
-    @JsonProperty( value = Constants.PREFIX,required = false)
+    @JsonProperty( value = Constants.PREFIX, required = false )
     public String getPrefix() {
         return prefix;
     }
 
-    @Override public Object apply( String o ) {
+    @Override
+    public Object apply( String o ) {
+        if ( StringUtils.isBlank( o ) ) {
+            return "";
```

Comment:
shouldn't we return null here? so that we're not writing empty strings to the backend (unless we want empty strings?)